adoc/chapters/programming_interface.adoc

```diff
@@ -2248,8 +2271,14 @@ info::device::atomic_memory_order_capabilities
 ----
 
     @ [.code]#std::vector<memory_order>#
-   a@ Returns the set of memory orderings supported by atomic operations on the device,
-      which is guaranteed to include [code]#relaxed#.
+   a@ Returns the set of memory orders supported by atomic operations on the
+      device.  When a device returns a "stronger" memory order in this set, it
+      must also return all "weaker" memory orders.  The memory orders
+      [code]#memory_order::acquire#, [code]#memory_order::release#, and
+      [code]#memory_order::acq_rel# are all the same strength.  If a device
+      returns one of these, it must return them all.
```

Comment:
As I said, would it be possible to have one device having only acquire while another one only release?

Reply:
This query (`info::device::atomic_memory_order_capabilities`) tells the operations that are supported within a single device, not between different devices.  Within a single device, it does not make sense to have "acquire" without "release".

The query `info::context::atomic_memory_order_capabilities` tells the capabilities between devices.  I agree that this query cannot express the situation where one device can do only "acquire" operations while another device can do only "release" operations.  We would need some completely different query to express this, though.  The existing `info::context::atomic_memory_order_capabilities` query can only describe operations that are allowed across a set of devices.  It cannot describe a situation where each device has a different set of allowable operations.  Fixing this is beyond the scope of this PR.